dependencies.yml

```diff
@@ -59,11 +59,11 @@ com.google.guava:
       to: com.linecorp.armeria.internal.shaded.guava
 
 com.google.protobuf:
-  protoc: { version: '3.5.1-1' }
-  protobuf-gradle-plugin: { version: '0.8.5' }
+  protoc: { version: '3.6.0' }
```

Comment:
`protoc` should match the version of `protobuf` since it's possible for their to be backwards incompatible changes in a compiler. Since we manage `protoc`, maybe we want to manage `protobuf` as well, currently it's whatever gRPC uses, which is at 3.5.1 even at the latest version.

Reply:
Thanks! Reverted.